README.md

```diff
@@ -10,6 +12,16 @@ To compile, you need the `magic` version of the [Rock compiler](https://github.c
 
 Tests can be run using the `test.sh` script.
 
+### Dependencies
+The following software is necessary to build version X.X.X of ooc-kean:
+* [rock X.X.X](https://github.com/magic-lang/rock/releases/tag/rock_X.X.X)
+* gcc 4.8.4
+
+### Microsoft Windows
+Rock have been built and tested on the following platforms:
+* Built with `MinGW` on Windows 7
```

Comment:
Maybe write that it's been tested on Ubuntu 14.04 and Windows 7. The rock repo has further details on how to compile on the two platforms.